Sources/Danger/Plugins/SwiftLint/SwiftLint.swift

```diff
@@ -144,20 +206,17 @@ extension SwiftLint {
     }
 
     // swiftlint:disable function_parameter_count
-    private static func lintModifiedAndCreated(danger: DangerDSL,
-                                               directory: String?,
-                                               arguments: [String],
-                                               shellExecutor: ShellExecuting,
-                                               swiftlintPath: String,
-                                               outputFilePath: String,
-                                               failAction: (String) -> Void,
-                                               readFile: (String) -> String) -> [SwiftLintViolation] {
-        // Gathers modified+created files, invokes SwiftLint on each, and posts collected errors+warnings to Danger.
-        var files = (danger.git.createdFiles + danger.git.modifiedFiles).filter { $0.fileType == .swift }
-        if let directory = directory {
-            files = files.filter { $0.hasPrefix(directory) }
-        }
+    private static func lintFiles(_ files: [File],
+                                  danger: DangerDSL,
+                                  arguments: [String],
+                                  shellExecutor: ShellExecuting,
+                                  swiftlintPath: String,
+                                  outputFilePath: String,
+                                  failAction: (String) -> Void,
+                                  readFile: (String) -> String) -> [SwiftLintViolation] {
 
+        let files = files.filter { $0.fileType == .swift }
+        
```

Comment:
<!--
  0 failure: 
  1 warning:  Lines should not ...
  
  
  DangerID: danger-id-Danger;
  File: Sources/Danger/Plugins/SwiftLint/SwiftLint.swift;
  Line: 219;
-->

- :warning: Lines should not have trailing whitespace. (`trailing_whitespace`)